recipes/ambertools/build.sh

```diff
@@ -0,0 +1,218 @@
+# cp -rf $SRC_DIR/include/fftw3.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf.inc $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/defreal.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_mem.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/sff.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/sander.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/nab_netcdf.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/fftw3.f $PREFIX/include/
+# cp -rf $SRC_DIR/include/nabtypes.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_nf_data.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_fortv2_c_interfaces.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_nc_data.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_nc_interfaces.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/nab.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/mdgx.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/CompatibilityMacros.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/xmin_opt.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/sander_api.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/nabcode.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/sanderles_api.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_nf_interfaces.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_f03.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/typesizes.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf.mod $PREFIX/include/
+# cp -rf $SRC_DIR/include/boost $PREFIX/include/
+# cp -rf $SRC_DIR/include/netcdf_meta.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/lmod_opt.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/AmberNetcdf.h $PREFIX/include/
+# cp -rf $SRC_DIR/include/fftw3.f03 $PREFIX/include/
+# cp -rf $SRC_DIR/dat/reslib $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/mmpbsa $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/pymsmt $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/slko $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/map.NEF-AMBER $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/reduce_wwPDB_het_dict.txt $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/PROTON_INFO $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/chamber $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/antechamber $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/Karplus.txt $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/CMakeLists.txt $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/dgdb $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/rism3d $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/contrib $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/pixmap $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/rism1d $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/charmmlipid2amber $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/map.DG-AMBER $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/leap $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/XaLeap_wcl $PREFIX/dat/
+# cp -rf $SRC_DIR/dat/PROTON_INFO.pdbv2 $PREFIX/dat/
+
+mkdir build
+cd build
+cmake  -DCMAKE_INSTALL_PREFIX=${PREFIX} -D CMAKE_C_COMPILER=${CC} -D CMAKE_CXX_COMPILER=${CXX} -DBUILD_GUI=FALSE -DOPENMP=TRUE -DPYTHON_EXECUTABLE=${PREFIX}/bin/python -DCOMPILER=GNU -DDOWNLOAD_MINICONDA=FALSE -DINSTALL_TESTS=TRUE "-DDISABLE_TOOLS=" ..
```

Comment:
```suggestion
cmake  -DCMAKE_INSTALL_PREFIX=${PREFIX} -D CMAKE_C_COMPILER=${CC} -D CMAKE_CXX_COMPILER=${CXX} -DBUILD_GUI=FALSE -DOPENMP=TRUE -DPYTHON_EXECUTABLE=${PREFIX}/bin/python -DCOMPILER=MANUAL -DDOWNLOAD_MINICONDA=FALSE -DINSTALL_TESTS=TRUE "-DDISABLE_TOOLS=" ..
```

Reply:
MANUAL uses environment variables or CMAKE variables.

Reply:
@njzjz I was not able to got the CMAKE setup to work. But I was partially lucky with autotools. Are you interested in the latest version, that actually builds? Then I can push it. Any help is really welcome :)

Reply:
According to [AmberCompilerConfig.cmake](https://github.com/Amber-MD/cmake-buildscripts/blob/master/AmberCompilerConfig.cmake), if you want to use CMAKE variables such as CMAKE_C_COMPILER, you need to set COMPILER value to MANUAL. When setting it to GNU it just use `gcc`, which is not installed.